Fix build with uClibc-ng

The elf.h header in uClibc-ng is missing the AT_HWCAP2 definition. Add it in
the code.

Signed-off-by: Baruch Siach <baruch@tkos.co.il>
---
Upstream status: Not upstreamable; uClibc needs to update elf.h

diff -Nuar nss-3.38.orig/nss/lib/freebl/blinit.c nss-3.38/nss/lib/freebl/blinit.c
--- nss-3.38.orig/nss/lib/freebl/blinit.c	2018-06-21 12:24:45.000000000 +0300
+++ nss-3.38/nss/lib/freebl/blinit.c	2018-06-26 13:13:55.636434720 +0300
@@ -100,6 +100,9 @@
     defined(__GNUC__) && __GNUC__ >= 2 && defined(__ELF__)
 #include <sys/auxv.h>
 extern unsigned long getauxval(unsigned long type) __attribute__((weak));
+#ifndef AT_HWCAP2
+#define AT_HWCAP2	26
+#endif
 #else
 static unsigned long (*getauxval)(unsigned long) = NULL;
 #define AT_HWCAP2 0
